gamemodes/terrortown/entities/weapons/weapon_zm_improvised.lua

```diff
@@ -108,7 +108,7 @@ function SWEP:OpenEnt(hitEnt)
                 hitEnt:Fire("Unlock", nil, 0)
             end
 
-            if unlock or hitEnt:HasSpawnFlags(SF_NPC_LONG_RANGE) then -- Long Visibility/Shoot
+            if unlock or hitEnt:HasSpawnFlags(256) then
```

Comment:
Is there a reason you used this magic number instead of a spawn flag?

Reply:
the old one is intended for npcs, it makes zero sense to use it for door entities

Reply:
ah, yes, that makes sense!

Reply:
And I was about to suggest a comment, but you already did it. Thank you